src/app/content/studyGuides/reducer.ts

```diff
@@ -30,9 +31,17 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
   switch (action.type) {
     case getType(locationChange): {
       const summaryShouldBeOpen = action.payload.query[modalQueryParameterName] === modalUrlName
-        && action.payload.action === 'PUSH';
+        && (action.payload.action === 'PUSH' || action.payload.action === 'REPLACE');
+      const colors = getColorFiltersFromQuery(action.payload.query) || state.summary.filters.colors;
 
-      return {...state, summary: {...state.summary, open: summaryShouldBeOpen}};
+      return {
+        ...state,
+        summary: {
+          ...state.summary,
+          filters: {...state.summary.filters, colors},
```

Comment:
we want to remove the reduction of `updateSummaryFilters` below, and rely on this instead